.github/workflows/build.yml

```diff
@@ -0,0 +1,87 @@
+name: Go CI
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  build:
```

Comment:
Build job should run for every commit, not just for commits of the  `dev` branch